<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>417</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>799</width>
    <height>417</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>799</width>
    <height>417</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>输入法演示窗口</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>F:/个性化/710枚白色图标/bdllq.ico</normaloff>F:/个性化/710枚白色图标/bdllq.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	background:url(&quot;background.JPG&quot;);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget{
	background:url(&quot;background.jpg&quot;);
}</string>
   </property>
   <widget class="QLabel" name="inputDisplayLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#inputDisplayLabel{
	color:white;
	font-size:20px;
	background:rgb(199, 199, 199, 40%);
	border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>631</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#textEdit{
	border-radius:50px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#statusLabel{
	color:white;
}</string>
    </property>
    <property name="text">
     <string>CN</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="capsLock">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>240</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#capsLock{
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Caps Off</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="wordLabel_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#wordLabel_1{
	background:rgb(138,138,138,0%);
	font-size:25px;
	border-radius:5px;	
	color:white;
}</string>
    </property>
    <property name="text">
     <string>1.</string>
    </property>
   </widget>
   <widget class="QLabel" name="wordLabel_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#wordLabel_2{
	background:rgb(138,138,138,0%);
	font-size:25px;
	border-radius:5px;	
	color:white;
}</string>
    </property>
    <property name="text">
     <string>2.</string>
    </property>
   </widget>
   <widget class="QLabel" name="wordLabel_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#wordLabel_3{
	background:rgb(138,138,138,0%);
	font-size:25px;
	border-radius:5px;	
	color:white;
}</string>
    </property>
    <property name="text">
     <string>3.</string>
    </property>
   </widget>
   <widget class="QLabel" name="wordLabel_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#wordLabel_4{
	background:rgb(138,138,138,0%);
	font-size:25px;
	border-radius:5px;	
	color:white;
}</string>
    </property>
    <property name="text">
     <string>4.</string>
    </property>
   </widget>
   <widget class="QLabel" name="wordLabel_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#wordLabel_5{
	font-size:25px;
	border-radius:5px;	
	color:white;
}</string>
    </property>
    <property name="text">
     <string>5.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
